README.md

```diff
@@ -32,6 +32,52 @@ Additionally, note that SCOPFLOW and SOPFLOW with HiOp solver use Ipopt to solve
 
 Details installation instructions are given at [INSTALL.md](./INSTALL.md) for information on acquiring, building and installing ExaGO.
 
+If you are a developer with access to the project, we also provide public binaries that are generated through our GitHub actions workflows documented in [README.md](.github/workflows/README.md), and with documentation about usage in the packages section of our repository. Check out a short (< 60s demo) of pulling down a version of ExaGO:
+
+[![asciicast](
+https://asciinema.org/a/KCi5TmUXc6zWDj7JYHzfSFxmw.png)](
+https://asciinema.org/a/KCi5TmUXc6zWDj7JYHzfSFxmw)
+
+### Commands used
+
+1. `$ export CR_PAT=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`
+    * https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens
+    * Make a PAT that is read-only for the ExaGO package registry
+    * You need to be a trusted collaborator in order to pull/push from the ghcr registry
+1. `$ echo $CR_PAT | docker login ghcr.io -u USERNAME --password-stdin`
+    * https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-with-a-personal-access-token-classic
+    * `USERNAME` is a placeholder for your username. Since we are using a PAT, it actulaly doens't matter what username is used
+    * If your PAT is ever exposed over plain text, make sure to re-generate a new one
+1. `$ docker pull ghcr.io/pnnl/exago:exago-develop-x436raytwpfhkihapsbpto3jd3dpfyqg.spack`
+    * https://github.com/pnnl/ExaGO/pkgs/container/exago
+    * Our ghcr package registry has ExaGO, along with base images and associated dependencies
+    * Instructions for using packages as base images or standalone containers are available for copying in GitHub website
+1. `$ docker run -it ghcr.io/pnnl/exago:exago-develop-x436raytwpfhkihapsbpto3jd3dpfyqg.spack`
+    * Run container interactively to try and run ExaGO
+1. `$ opflow --version`
+    * Test that opflow CLI application is available
+1. `$ python -c "import exago"`
+    * Test importing python wrapper
+
+## Developer Guide
+
+You can view the following helpful documentation sources:
+- [test_add.md](docs/web/test_add.md) markdown file for information on adding tets (outdated)
+- [README.md](buildsystem/README.md) for our bash / spack buildsystem used in GitHub/GitLab CI/CD
+- [README.md](buildsystem/spack/README.md) for our spack specific build scripts that support CI tcl modules on HPC target platforms
+- [README.md](docs/devcontainer/README.md) for our devcontianer configuration information (codespace support coming soon)
```

Comment:
@jaelynlitz devcontainer docs are linked here - do you want to move/re-link in tutorials section as well?

Reply:
yeah maybe re-link. my thought process was if I just scrolled to the tutorials header, what do I need to get started?